en/ros2/user_guide.md

```diff
@@ -464,18 +464,18 @@ Use the following commands to install the correct ROS 2/gz interface packages (n
 :::: tabs
 
 ::: tab humble
-To install the bridge for use with ROS 2 "Humble" and Gazebo Garden (on Ubuntu 22.04):
+To install the bridge for use with ROS 2 "Humble" and Gazebo Harmonic (on Ubuntu 22.04):
 
 ```sh
-sudo apt install ros-humble-ros-gzgarden
+sudo apt install ros-humble-ros-gzharmonic
 ```
 
 :::
 
 ::: tab foxy
-First you will need to [install Gazebo Garden](../sim_gazebo_gz/index.md#installation-ubuntu-linux), as by default Foxy comes with Gazebo Classic 11.
+First you will need to [install Gazebo Garden](../sim_gazebo_gz/index.md#installation-ubuntu-linux), as by default Foxy comes with Gazebo Classic 11. <!-- note, garden is EOL Nov 2024 -->
```

Comment:
This whole section needs to be discussed - "what to do with Foxy when Foxy doesn't support Harmonic". I tend to think delete this all and we'll only use harmonic in next version.

Reply:
I agree with you; we should encourage people to use the latest version of Gazebo, which is Harmonic, as recommended by the Gazebo wiki. The Garden version is not fully stable for ROS distributions.

Here's the link for reference: [Gazebo ROS Installation](https://gazebosim.org/docs/latest/ros_installation/).

Reply:
Thanks. I'm hoping for confirmation from @beniaminopozzan, because sometimes with these things there are long term users that rely on particular variants, or other reasons. For example, if a ROS variant couldn't work on a Ubuntu version we are required to support.

Reply:
Thanks, that is what I was thinking.

We should certainly encourage the upgrade. You can still se Gazebo Classic 11 on Ubuntu 22.04 (by installing separately) - but can you use it **with Harmonic**?

I will think on this, but I lean towards dropping the Garden entirely. I'll come up with something next week.

Reply:
do you mean Gazebo Classic 11 together with Gazebo Harmonic? No, they are not compatible just like Gazebo Classic 11 and Gazebo Garden.